rwslib/rws_requests/__init__.py

```diff
@@ -525,3 +525,47 @@ def url_path(self):
             args.append(self.formoid)
 
         return make_url(*args, **self._querystring())
+
+
+class ConfigurableDatasetRequest(RWSAuthorizedGetRequest):
+    VALID_DATASET_FORMATS = ()
```

Comment:
Intent of this is to suit subclassing of this; we subclass this for a particular Configurable Dataset, set the class attribute and an any instance of the subclass will check the dataset_format against the `VALID_DATASET_FORMATS`.  I deliberately went with a tuple, because it is non-mutable.